src/main/java/games/strategy/engine/framework/startup/mc/ServerConnectionProps.java

```diff
@@ -0,0 +1,46 @@
+package games.strategy.engine.framework.startup.mc;
+
+/**
+ * Simple data class for parameter values to connect to a remote host server (EG: to connect to a hosted game).
+ */
+public class ServerConnectionProps {
+  /** Player name, the desired name for the current player connecting to remote host. */
+  private String name;
+  /** Remote host address. */
+  private String host;
+  private int port;
+  /** Password to use to connect to the remotely hosted game, this is set by the host of the game. */
+  private String password;
+
+  public String getHost() {
+    return host;
+  }
+
+  public void setHost(final String host) {
```

Comment:
I'd suggest making this class immutable if possible and using a single constructor instead of setters